File: AirTrafficControler/src/FlightCoordination/Placement.java
package FlightCoordination;

public class Placement implements Comparable<Placement>{
	String	   pid; //Placement Identifier.
	FlightPlan fp ; //Flight plan for the placement.
	
	public int compareTo(Placement other){
		return this.pid.compareTo(other.getPid());
	}
	//public Placement (String pid){
	//	this.pid = pid;
	//}
	public Placement (String pid, FlightPlan fp){
		this.pid = pid;
		this.fp  = fp;
	}
	public FlightPlan setFp(FlightPlan fp){
		if (fp == null){
			throw new AssertionError();
		}
		return this.fp = fp;
	}
	public String getPid(){
		return this.pid;
	}
	public FlightPlan getFp(){
		return this.fp;
	}
	public void addFlight(Flight f) {
		this.fp.addFlight(f);
	}
	public int size() {
		return fp.size();
	}
}

File: AirTrafficController_Python/Placement.py

from FlightPlan import FlightPlan


class Placement:
    def __init__(self, pid, fp):
        self.pid = pid
        self.fp  = FlightPlan(fp)

    def compareTo(self, other):
        return (self.pid > other.pid) - (self.pid < other.pid)

    def setFp(self, fp):
        if fp == None:
            raise AssertionError("FlightPlan can't be None!")
        self.fp = FlightPlan(fp)

    def getPid(self):
        return self.pid

    def getFp(self):
        return self.fp

    def addFlight(self, f):
        return self.fp.addFlight(f)

    def size(self):
        return self.fp.size()

    def get(self, index):
        return self.fp.get(index)
